.github/workflows/build_tvos.yml

```diff
@@ -0,0 +1,154 @@
+        required: true
+        type: string
+      unity_platform_name:
+        description: 'The platform name Unity should install with'
+        default: ''
+        required: true
+        type: string
+
+jobs:
+  build_tvos:
+    name: build-tvos-unity${{ inputs.unity_version }}-CPP${{ inputs.firebase_cpp_sdk_version }}
+    runs-on: macos-12
+    strategy:
+      fail-fast: false
+
+    env:
+      # LC_ALL, LANG and U3D_PASSWORD are needed for U3D.
+      LC_ALL: en_US.UTF-8
+      LANG: en_US.UTF-8
+      U3D_PASSWORD: ""
+      # Disable checking for U3D updates, since it is buggy
+      U3D_SKIP_UPDATE_CHECK: 1
+      xcodeVersion: "13.3.1"
+
+    steps:
+      - name: Checkout Unity Repo
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          ref: ${{ inputs.unity_branch }}
+
+      - name: Checkout CPP Repo
+        uses: actions/checkout@v2
+        with:
+          repository: firebase/firebase-cpp-sdk
+          path: firebase-cpp-sdk
+          ref: ${{ inputs.firebase_cpp_sdk_version }}
+          submodules: true
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+
+      - name: setup Xcode version
+        run: sudo xcode-select -s /Applications/Xcode_${{ env.xcodeVersion }}.app/Contents/Developer
+
+      - name: Install prerequisites
+        shell: bash
+        run: |
+          echo "FIREBASE_CPP_SDK_DIR=${{ github.workspace }}/firebase-cpp-sdk" >> $GITHUB_ENV
+          cd firebase-cpp-sdk
+          python scripts/gha/install_prereqs_desktop.py
+          cd ..
+
+      - name: Install Unity installer (U3D)
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 10
+          max_attempts: 3
+          shell: bash
+          command:  gem install u3d -v 1.2.3
+
+      - name: Install python deps
+        shell: bash
+        run: |
+          pip install -r scripts/gha/requirements.txt
+
+      - name: Install Unity
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 15
+          max_attempts: 3
+          shell: bash
+          command:  |
+            python scripts/gha/unity_installer.py --install --platforms ${{ inputs.unity_platform_name }} --version ${{ inputs.unity_version }}
+
+      - name: Setup Unity path
+        shell: bash
+        run: |
+          echo "UNITY_ROOT_DIR=$( python scripts/gha/print_matrix_configuration.py -u ${{ inputs.unity_version }} -k unity_path )" >> $GITHUB_ENV
+
+      - name: Build SDK (tvOS)
+        timeout-minutes: 90
+        shell: bash
+        run: |
+          # TODO add handling cmake_extras
+          python scripts/build_scripts/build_zips.py --gha --platform=tvos --unity_root=$UNITY_ROOT_DIR --apis=${{ inputs.apis }}
+
+      - name: Check zip file
+        shell: bash
+        run: |
+          if [ -f ios_unity/*.zip ]; then
```

Comment:
is the build_zip python script update to create tvos_unity folder? Not sure i've seen the related PR yet.

Reply:
Yep!

Reply:
Great, if that's the case the check needs to update to if [ -f tvos_unity/*.zip ]